assets/src/edit-story/panels/panel/title.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import panelContext from './context';
+import DragHandle from './handle';
+import Arrow from './arrow.svg';
+
+const Header = styled.header`
+	background-color: ${ ( { theme, isPrimary } ) => isPrimary ? theme.colors.fg.v2 : theme.colors.fg.v1 };
+	border: 0px solid ${ ( { theme } ) => theme.colors.fg.v2 };
+	border-top-width: ${ ( { isPrimary } ) => isPrimary ? 0 : '1px' };
+	color: ${ ( { theme } ) => theme.colors.bg.v2 };
+	display: flex;
+	padding: 10px 20px;
+	justify-content: space-between;
+	align-items: center;
+	position: relative;
+`;
+
+const H = styled.h1`
```

Comment:
Minor nitpick: What about `Heading` instead of `H`?

Reply:
Alright, now we have both `Title`, `Header` and `Heading` inside the same file. What could go wrong? 😝

Fixed in a2443f8ab.